File: docs/components_page/components/layout/width.R
library(dashBootstrapComponents)

row <- div(
  list(
    dbcRow(
      dbcCol(div("A single, half-width column"), width = 6)
    ),
    dbcRow(
      dbcCol(div("An automatically sized column"), width = "auto")
    ),
    dbcRow(
      list(
        dbcCol(div("One of three columns"), width = 3),
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns"), width = 3)
      )
    )
  )
)

File: docs/components_page/components/layout/width.py
import dash_bootstrap_components as dbc
from dash import html

row = html.Div(
    [
        dbc.Row(dbc.Col(html.Div("A single, half-width column"), width=6)),
        dbc.Row(
            dbc.Col(html.Div("An automatically sized column"), width="auto")
        ),
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns"), width=3),
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns"), width=3),
            ]
        ),
    ]
)
